content/blog/2022/dependabot-single-pull-request-python.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Replace Dependabot With a Single Dependency Upgrade Pull Request
+author: ed
+date: 2022-05-27
+tags:
+  - Article
+  - Build Tools
+  - DevOps
+  - JavaScript
+  - Pip
+  - Python
+  - Tips
+image:
+  src: blog/2022/bot.jpg
+summary: |
+  Keeping your project dependencies up to date can become an unwieldy task when
+  Dependabot opens a dozen pull requests every week. We present an alternative
+  approach that uses only one scheduled pull request for all upgrades.
+---
+
+[Dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates)
+is a useful tool provided by GitHub to all repositories. Maintainers only need
+to specify the frequency at which they want to check for updates and Dependabot
+will open a pull request for each package that has published a new version since
+the last check. Combined with a robust automated test suite, this greatly
+reduces the burden of manually keeping dependencies up to date.
+
+However, as projects grow and more dependencies are added, the noise produced by
+individual pull requests starts becoming an issue in itself. That's why a
```

Comment:
Minor: I think you could say either  `starts becoming an issue itself` or `starts becoming an issue in an of itself`

Reply:
I believe Jonny added `in itself` in a [previous commit](https://github.com/oddbird/oddleventy/pull/210/commits/c2f35eb8078a8e1fbc0afd5443644ab401faa166#diff-9a9ba9628761ab4edc1ccb8687aac0f12971fdf6ea2132dfaa6b287854487d46R28), so I don't know which is correct 😅